evalml/model_understanding/graphs.py

```diff
@@ -564,8 +566,12 @@ def partial_dependence(pipeline, X, features, grid_resolution=100):
     if ((isinstance(features, int) and X.iloc[:, features].isnull().sum()) or (isinstance(features, str) and X[features].isnull().sum())):
         warnings.warn("There are null values in the features, which will cause NaN values in the partial dependence output. Fill in these values to remove the NaN values.", NullsInColumnWarning)
 
+    if ((isinstance(features, int) and X.iloc[:, features].value_counts(normalize=True).values[0] + 0.01 > percentiles[1]) or
+       (isinstance(features, str) and X[features].value_counts(normalize=True).values[0] + 0.01 > percentiles[1])):
+        raise ValueError(f"Feature {features} is mostly one value and cannot be used to compute partial dependence. Try raising the upper percentage value.")
```

Comment:
Two questions:
- `Features {features}` will return a number if `features` is an int right? Should we return the name of the feature in this case for easier debugging?
- Should we include some information about the value causing the imbalance in the error message like `The value {X.iloc[:, features].value_counts(normalize=True).values[0]} is present in feature {features} above the upper limit for partial dependence..etc`?

Reply:
Ah good catch in that first bullet. Since the `features` variable is passed by the user, I assumed it would be fine to output that same value, but I'll add some quotations around it to make it clearer to the user.

I can add in a small clarification on the first value that's causing this

Reply:
@ParthivNaresh I added a fix, let me know if it looks good to you!